source/help/apps/desktop-changelog.rst

```diff
@@ -6,6 +6,9 @@ Release v4.6
 
 **Download Binaries:** `Mattermost Desktop on GitHub <https://github.com/mattermost/desktop/releases/tag/v4.6.1>`_
 
+
+- **v4.6.2, release day TBD**
+ - Investigating fixing an issue where logging in to ``gitlab.com`` does not work on the desktop apps. `MM-31626 <https://mattermost.atlassian.net/browse/MM-31626>`_
```

Comment:
```suggestion
 - Investigating fixing an issue where logging in to ``gitlab.com`` does not work on the Desktop App. `MM-31626 <https://mattermost.atlassian.net/browse/MM-31626>`_
```

Reply:
Is this issue present on a specific version(s) of the Desktop App (such as 4.6.1)? If so, consider adding that detail to the bullet point as well. If adding version details, you can omit "the" -- example: "does not work on Desktop App [version]."

Reply:
This issue affects all versions.